src/helpers/util.js

```diff
@@ -0,0 +1,64 @@
+let STANDARD_ASPECT_RATIOS = [
+  [1, '1:1'],
+  [4 / 3, '4:3'],
+  [5 / 4, '5:4'],
+  [3 / 2, '3:2'],
+  [16 / 10, '16:10'],
+  [16 / 9, '16:9'],
+  [21 / 9, '21:9'],
+  [32 / 9, '32:9'],
+]
+let RATIOS = STANDARD_ASPECT_RATIOS.map(function (tpl) { return tpl[0] }).sort()
+let LOOKUP = Object()
+for (let i = 0; i < STANDARD_ASPECT_RATIOS.length; i++) {
+  LOOKUP[STANDARD_ASPECT_RATIOS[i][0]] = STANDARD_ASPECT_RATIOS[i][1]
+}
+
+/*
+Find the closest value in a sorted array
+*/
+function findClosest(arrSorted, value) {
+  let closest = arrSorted[0]
+  let closestDiff = Math.abs(arrSorted[0] - value)
+  for (let i = 1; i < arrSorted.length; i++) {
+    let diff = Math.abs(arrSorted[i] - value)
+    if (diff < closestDiff) {
+      closestDiff = diff
+      closest = arrSorted[i]
+    } else {
+      return closest
+    }
+  }
+  return arrSorted[arrSorted.length - 1]
+}
+
+/*
+Estimate the aspect ratio based on width x height
+*/
+export function estimateAspectRatio(width, height, errorAllowed, aspectRatio) {
+  let ratio = Math.max(width, height) / Math.min(width, height)
```

Comment:
Hey, the previous comment still holds true here, we will have to remove the max and min functions because they kind-of always divide the greater dimension with the smaller one. Let it always be `ratio = width / height`. This helps us define exact aspect ratios - including orientation 😄

Reply:
yeah 🙈